angular.json

```diff
@@ -28,9 +29,11 @@
                 "glob": "**/*.svg",
                 "input": "node_modules/ionicons/dist/ionicons/svg",
                 "output": "./svg"
-              }
+              },
+              "src/_redirects"
```

Comment:
Added this according to Vitor's recommendation, it's a Netlify redirect configuration